celsos_bookSearch/celsos_bookSearch/Model/ServiceErrorEnum.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  ServiceErrorEnum.swift
+//  celsos_bookSearchTests
+//
+//  Created by Celso Junio Simões de Oliveira Santos on 18/06/21.
+//
+
+import Foundation
+// Enum to Simulate Promise Errors
+public enum ServiceError: Error {
+    case urlMalFormatted
```

Comment:
Aparentemente vc optou por colocar os nomes das var e let em inglês, então é melhor tirar esse "mal" e colocar em inglês XD

Reply:
My bad, o correto é `malformed` se for inglês :D